packages/core/src/App/Containers/WarningScamMessageModal/warning-scam-message.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import { Modal, DesktopWrapper, MobileWrapper } from '@deriv/components';
+import { PropTypes } from 'prop-types';
+import { withRouter } from 'react-router';
+import './warning-scam-message-modal.scss';
+import WarningScamMessageModal from './warning-scam-message-modal';
+
+const WarningScamMessage = ({ is_dark_mode_on, is_warning_scam_message_modal_visible, setScamMessageLocalStorage }) => {
+    const [is_message_read, setIsMessageRead] = React.useState(false);
+
+    const acknowledgeMessage = () => setIsMessageRead(!is_message_read);
+
+    /* Check whether the account is logged_in, and client from country 'br' = Brazil, and have not yet read the scam message warning pop up */
+    if (is_warning_scam_message_modal_visible) {
+        return (
+            <React.Fragment>
+                <DesktopWrapper>
+                    <Modal
+                        id='warning_scam_message_modal'
+                        className='warning-scam-message-modal'
+                        is_open={is_warning_scam_message_modal_visible}
+                        small
+                    >
+                        <WarningScamMessageModal
+                            acknowledgeMessage={acknowledgeMessage}
+                            setLocalStorage={setScamMessageLocalStorage}
+                            is_message_read={!is_message_read}
+                            is_dark_mode_on={is_dark_mode_on}
+                        />
+                    </Modal>
+                </DesktopWrapper>
+                <MobileWrapper>
+                    <Modal
+                        id='warning_scam_message'
+                        className='warning-scam-message'
+                        is_open={is_warning_scam_message_modal_visible}
+                        small
+                    >
+                        <WarningScamMessageModal
+                            acknowledgeMessage={acknowledgeMessage}
+                            setLocalStorage={setScamMessageLocalStorage}
+                            is_message_read={!is_message_read}
+                        />
+                    </Modal>
+                </MobileWrapper>
+            </React.Fragment>
+        );
+    }
+    return null;
+};
+
+WarningScamMessage.propTypes = {
+    displayWarningScamMessage: PropTypes.func,
+    is_dark_mode_on: PropTypes.bool,
```

Comment:
it seems we don’t use displayWarningScamMessage anywhere... could you check it please

Reply:
Yea, will remove it from the files. Thanks @yauheni-kryzhyk-deriv 🙏